e2e/test/iothub/service/QueryClientE2ETests.cs

```diff
@@ -132,23 +136,36 @@ public async Task JobQuery_QueryWorks()
             queriedJob.JobType.Should().NotBeNull();
             queriedJob.CreatedTimeUtc.Should().NotBeNull();
             queriedJob.Status.Should().NotBeNull();
+            if (queriedJob.Status != JobStatus.Queued
+                && queriedJob.Status != JobStatus.Scheduled
+                && queriedJob.Status != JobStatus.Cancelled
+                && queriedJob.Status != JobStatus.Unknown)
+            {
+                queriedJob.StartTimeUtc.Should().NotBeNull();
```

Comment:
Optionally check these values when they should be there, so we know we are deserializing them fine.